exasol/analytics/audit/audit_query_handler.py

```diff
@@ -121,8 +141,8 @@ def _final_continue(self, audit_query: AuditQuery) -> Continue:
         Create Continue for excecuting the final AuditQuery.
         """
         input_query = SelectQueryWithColumnDefinition(
-            query_string="SELECT 1",
-            output_columns=[],
+            query_string="SELECT 1 as DUMMY_COLUMN",
+            output_columns=[decimal_column("DUMMY_COLUMN", precision=10)],
```

Comment:
```suggestion
            output_columns=[decimal_column("DUMMY_COLUMN", precision=1, scale=0)],
```

Reply:
see next push - I applied this change manually to avoid merge conflicts.